packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -1504,6 +1532,39 @@ export interface DestroyOptions {
   readonly ci?: boolean;
 }
 
+/**
+ * Options for the garbage collection
+ */
+export interface GarbageCollectionOptions {
+  /**
+   * The action to perform.
+   *
+   * @default 'full'
+   */
+  readonly action: 'print' | 'tag' | 'delete-tagged' | 'full';
+
+  /**
+   * The type of the assets to be garbage collected.
+   *
+   * @default 'all'
+   */
+  readonly type: 's3' | 'ecr' | 'all';
```

Comment:
IMO lets make this an enum now. Won't really hurt anything to just make this a string enum
```suggestion
enum GCType{
  S3 = 's3',
  ECR = 'ecr',
  ALL = 'all'
}
```

Reply:
Personally I don't feel this is necessary.

Reply:
I agree, so I'm gonna not do it. The values we get are strings so it seems unnecessary to cast into enums.